network_simulator/device.py

```diff
@@ -0,0 +1,79 @@
+"""This module contains all device definitions.
+
+.. autosummary::
+    Host
+    Router
+"""
+
+# Imports go here.
+# e.g.    from .common import ACK_PACKET_SIZE_BITS
+
+class Device(object):
+    """Representation of a device (Host or Router).
+
+    :param str name: name of this device from the input file.
```

Comment:
just include the :ivar in the documentation, not the :param, as per your formatting in Router.